src/modules/plus/subtitles/subtitles.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
This file is duplicated in Shotcut. We could move this into the MLT framework - maybe as "mlt_subtitle". But it does not have an object - it is just functions to read srt files.

Also, in order to move it into the MLT framework, I would need to rewrite it as C - which I am willing to do.

Reply:
An SRT helper does not logically belong in the MLT framework. It could be a separate project, but then you would have more to manage. In case there are changes needed for Shotcut, I do not want to require a new version of MLT. Distros do not build Shotcut the way we do, and may use a slightly older version. They have more to manage. So, leave it duplicated as-is for now.